backend/server/importlib/backend.py

```diff
@@ -668,7 +669,7 @@ def lookupErrataSeverityId(self, erratum):
     def lookupEVRs(self, evrHash):
         sql = "select LOOKUP_EVR(:epoch, :version, :release) id from dual"
         h = self.dbmodule.prepare(sql)
-        for evr in sorted(evrHash.keys(), key=lambda x: (x[0] if x[0] != '' else None, x[1], x[2])):
+        for evr in sorted(evrHash.keys(), key=lambda k: (str(k or 0), k[1], k[2])):
```

Comment:
(k or 0) sounds wrong. Should it be `(k[0] or 0)` ?

Reply:
Right, addressed